solutions/swb-reference/integration-tests/support/complex/paabHelper.ts

```diff
@@ -24,14 +24,20 @@ export class PaabHelper {
   private _setup: Setup;
   private _randomTextGenerator: RandomTextGenerator;
   private _outputError: boolean | undefined;
+  private _numberOfProjects: number = 3;
 
-  public constructor(outputError?: boolean) {
+  public constructor(numberOfProjects?: number, outputError?: boolean) {
     this._setup = Setup.getSetup();
     this._randomTextGenerator = new RandomTextGenerator(this._setup.getSettings().get('runId'));
     this._outputError = outputError;
+    if (numberOfProjects) this._numberOfProjects = numberOfProjects;
   }
 
   public async createResources(): Promise<PaabResources> {
+    if (this._numberOfProjects > 3) {
+      throw new Error('PaabHelper cannot support more than 3 projects');
```

Comment:
What is the downside of allowing the creation of all resources every time? Is the cleanup method skipping resource deletion?